azkaban-exec-server/src/main/java/azkaban/execapp/AzkabanExecServerModule.java

```diff
@@ -29,12 +40,65 @@
  * structuring of Guice components.
  */
 public class AzkabanExecServerModule extends AbstractModule {
+
+  private static final int DEFAULT_THREAD_NUMBER = 50;
+  private static final int DEFAULT_HEADER_BUFFER_SIZE = 4096;
+  private static final int MAX_FORM_CONTENT_SIZE = 10 * 1024 * 1024;
+
+  private static final Logger logger = Logger.getLogger(AzkabanExecServerModule.class);
+
   @Override
   protected void configure() {
     bind(ExecutorLoader.class).to(JdbcExecutorLoader.class).in(Scopes.SINGLETON);
     bind(AzkabanExecutorServer.class).in(Scopes.SINGLETON);
     bind(TriggerManager.class).in(Scopes.SINGLETON);
     bind(FlowRunnerManager.class).in(Scopes.SINGLETON);
+  }
+
+  @Provides
+  @Named("ExecServer")
+  @Singleton
+  private Server createJettyServer(Props props) {
```

Comment:
Just like how AzkabanExecutorServer is constructed, can't the Server instance be constructed in the same way without using a Guice provider? 
I'd prefer keeping the module file as simple as possible.